spec/search-api.yaml

```diff
@@ -29,12 +29,12 @@ paths:
   /table/{table_name}/info:
     get:
       summary: Get a Table
-      description: Returns the Table identified by name.
+      description: Returns the information about the Table identified by name.
       operationId: getTable
       parameters:
         - name: table_name
           in: path
-          description: Unique Table name
+          description: Unique Table name, unique to this implementation
```

Comment:
This could still be clearer. If I understand the intent, maybe this is best:

```
description: The table name. Uniquely identifies the table within this API instance.
```